src/sider/senoppfolging/NewVurderingForm.tsx

```diff
@@ -13,7 +13,7 @@ const texts = {
   button: "Fullfør vurdering",
   description: "Når du fullfører vurderingen fjernes hendelsen fra oversikten.",
   begrunnelseDescription:
-    "Det er valgfritt å skrive begrunnelse. Begrunnelsen vil være synlig i Modia, men vises ikke for den sykmeldte. Begrunnelsen blir ikke journalført.",
+    "Det er valgfritt å skrive begrunnelse. Begrunnelsen vil være synlig i Modia, men vises ikke for den sykmeldte på innloggede sider. Innbyggeren kan kreve innsyn i det du skriver her. Begrunnelsen blir ikke journalført.",
```

Comment:
Er det nødvendig å skrive at det er valgfritt? 😊 Ellers i modia har vi pleid å skrive `Label (obligatorisk)` der vi krever et svar.
```suggestion
    "Begrunnelsen vil være synlig i Modia, men vises ikke for den sykmeldte på innloggede sider. Innbyggeren kan kreve innsyn i det du skriver her. Begrunnelsen blir ikke journalført.",
```

Reply:
Støtter forslaget, så blir det også litt kortere description :)

Reply:
Godt spørsmål! Jeg ser på det som et designvalg, og at det kunne vært fint å fulgt samme mønster som andre steder. Virker som vi anvender `label (obligatorisk)` og ingenting når det er frivillig? Automatisk ville jeg tenkt at hvis vi innfører en ny variant med `label (valgfritt)` ville den skapt en slags forskjell og usikkerhet rundt de feltene som er valgfrie uten en slik indikasjon. Men enig at det ble mye tekst og de sier det samme, så jeg bytter. Godt innspill! 😊